COMMUNITY-ROLES.md

```diff
@@ -138,21 +136,21 @@ using one of the template messages below as a base.
    go to https://github.com/orgs/tldr-pages/people
    and add them to the organization as a member.
 
-3. Open a PR moving their name to the "Organization members" section
+3. Open a PR adding their name to the "Organization members" section
    in [MAINTAINERS.md](MAINTAINERS.md#organization-members).
-   Make sure to include `Closes #<issue number>` in the PR description.
+   **Make sure to include `Closes #<issue number>` in the PR description.**
    The issue will then be automatically closed once the PR is merged.
 
 ### Adding new organization owners
 
-1. Open an issue with the following message template (edit it as appropriate):
+1. Open an issue with the following message template (editing as appropriate):
 
    ```
    Hi, @username! You've been an active tldr-pages organization member for over 6 months.
    Thanks for sticking around this far and helping out!
-   According to our [community roles documentation](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md), you're now eligible for becoming an owner in the organization.
+   According to our [community roles documentation](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md), you're now eligible to become an owner of the organization.
```

Comment:
I actually prefer the "owner in" rather than "owner of". I think it emphasises the "one among many" aspect rather than the "boss" aspect. I believe it is grammatically correct since "owner" is a specific name of a role that one can have in GitHub organizations, not a generic English term we're using in a sentence.